src/components/common/Menu.js

```diff
@@ -1,16 +1,46 @@
-import React from 'react';
-import { FormattedMessage } from 'react-intl';
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import routes from 'constants/routesPaths';
+import {
+  injectIntl,
+  intlShape,
+  defineMessages
+} from 'react-intl';
 
-const Menu = () => (
-  <div className="containerMenu">
-    <div className="hamburgerIcon" />
-    <div className="close" />
-    <ul className="menuList">
-      <li className="selected"> <FormattedMessage id="menu.about" /> </li>
-      <li> <FormattedMessage id="menu.contact" /> </li>
-    </ul>
-  </div>
-);
+const messages = defineMessages({
+  about: { id: 'menu.about' },
+  contact: { id: 'menu.contact' },
+});
 
-export default Menu;
+class Menu extends PureComponent {
+  static propTypes = {
+    show: PropTypes.bool,
+    intl: intlShape.isRequired,
+  };
 
+  render() {
+    const { show, intl } = this.props;
+
+    return (
+      <div className="containerMenu" style={{ display: show ? 'block' : 'none' }} >
```

Comment:
If you just set display to none, you'd be rendering the the tags, but they're hidden.

The idea is that you don't render what you don't need. To not render, just return null. You can do something like:

`if (!show) return null;`

Reply:
It sounds good. I think is more performant.